### ISSUE TITLE
machine specific dependsordered

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#357]
```
RDEPENDS:${PN} = "x y z"
RDEPENDS:${PN}:append:magic-machine = "a"
```

gives a dependsordered warning, but this should not report anything if the append is machine/package/a.s.o. specific
